src/web.security/middleware/index.ts

```diff
@@ -0,0 +1,80 @@
+import {NextFunction, Request, Response} from "express";
+import {COOKIE_NAME, OAUTH2_AUTH_URI, OAUTH2_CLIENT_ID,
+  OAUTH2_REDIRECT_URI} from "../../properties";
+import * as redisService from "../../services/redis.service";
+import * as keys from "../../session/keys";
+import logger from "../../logger";
+import {jweEncodeWithNonce, generateNonce} from "../jwt.encryption";
+
+const OAUTH_SCOPE_PREFIX = "https://api.companieshouse.gov.uk/company/";
+const COMPANY_NUMBER_KEY = "company_number";
```

Comment:
This is also newly defined in keys.ts above. Are they essentially the same thing and could you just import/reference that one here instead?